### PR TITLE
Status: 2023q3: SIMD: correction, specificity

### PR DESCRIPTION
**Pull request by grahamperrin**
Markup: a hard line break between the first and second links.

Manual page: link to the specified section (7).

Use 'either …, or …' within the second of four list items.

### CODE REVIEW

File: website/content/en/status/report-2023-07-2023-09/simd.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ The goal of this project is to provide SIMD-enhanced versions of common libc fun
 For each function optimised, up to four implementations will be provided:
 
  * a *scalar* implementation optimised for amd64, but without any SIMD usage,
- * a *baseline* implementation using SSE and SSE2 or alternatively an *x86-64-v2* implementation using all SSE extensions up to SSE4.2,
+ * either a *baseline* implementation using SSE and SSE2, or an *x86-64-v2* implementation using all SSE extensions up to SSE4.2,
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/simd.adoc:15]: This may be slightly easier to read in, for example, the plain text email.

**Comment by grahamperrin** [comment]
@clausecker for your consideration. Thanks.

**Comment by clausecker** [comment]
I used this phrasing because for all but two functions, it's likely going to be a baseline implementation and x86-64-v2 is a bit of a special case.  But if this wording feels awkward to a native speaker, sure, change it.

I don't think we need a `sektion=7` query parameter in the URL as there is no `simd` man page in any other section, but I guess it won't hurt.

LGTM.

**Comment by grahamperrin** [comment]
@clausecker thanks. 

Subtly: if `sektion=7` is omitted, then the title of the linked page will be _simd_, which is relatively vague for title bar, tab, bookmarking and other purposes.

An 'or alternatively' always catches my eye. Whilst grammatically correct, it's sometimes a trigger to look at surrounding texts.

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**